homeassistant/components/rotel/media_player.py

```diff
@@ -0,0 +1,493 @@
+"""
+Rotel platform.
+
+Although only the RSP-1570 is supported at the moment, the
+low level library could easily be updated to support other
+products of a similar vintage.
+"""
+import asyncio
+import logging
+
+import voluptuous as vol
+
+from homeassistant.components.media_player import (
+    DOMAIN, PLATFORM_SCHEMA, MediaPlayerDevice)
+from homeassistant.components.media_player.const import (
+    SUPPORT_SELECT_SOURCE, SUPPORT_TURN_OFF, SUPPORT_TURN_ON,
+    SUPPORT_VOLUME_MUTE, SUPPORT_VOLUME_SET, SUPPORT_VOLUME_STEP)
+from homeassistant.const import (
+    ATTR_ENTITY_ID, CONF_DEVICE, CONF_NAME, EVENT_HOMEASSISTANT_STOP,
+    STATE_OFF, STATE_ON)
+import homeassistant.helpers.config_validation as cv
+
+_LOGGER = logging.getLogger(__name__)
+
+DEFAULT_NAME = 'Rotel RSP-1570'
+
+SUPPORT_ROTEL_RSP1570 = \
+    SUPPORT_VOLUME_SET | SUPPORT_VOLUME_STEP | SUPPORT_VOLUME_MUTE | \
+    SUPPORT_TURN_ON | SUPPORT_TURN_OFF | SUPPORT_SELECT_SOURCE
+
+ROTEL_RSP1570_SOURCES = {
+    ' CD': 'SOURCE_CD',
```

Comment:
Extraneous space.

```suggestion
    'CD': 'SOURCE_CD',
```

Reply:
This is actually correct but I have decided to refactor the "third party" llbrary so that this nastiness isn't exposed to the user.   I'll flag this item as resolved and commit a cleaner version shortly.